src/index.html

```diff
@@ -7,8 +7,54 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="nav__wraper">
+      <a href="index.html">
+        <img
+          href="index.html"
+          class="nav__logo"
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav__nav">
+        <label for="ul">
```

Comment:
```suggestion
        <label for="ul">
```
лейбл тут не потрібний

Reply:
Remember that links should have cursor: pointer and clickable zone on **_100%_** of header height

Але як як я розумію ця вимога за допомогою паддінгів реалізується